### ISSUE TITLE
[bug] version field in generated pkg_config .pc files of components is None

### ISSUE THREAD

**Issue by madebr** [conan-io/conan#7630]
When using the pkg_config generator with a requirement using pkg_config, the generated .pc files contain `Version: None`.

### Environment Details (include every applicable attribute)
  * Operating System+version: Fedora Linux 30
  * Compiler+version: gcc 9
  * Conan version: 1.28.2
  * Python version: 3.7.7

### Steps to reproduce (Include if Applicable)

```sh
#!/bin/bash
mkdir -p test_package

cat >conanfile.py <<EOF
from conans import ConanFile
class A(ConanFile):
    name = "some_package"
    version = "0.1"
    def package_info(self):
        self.cpp_info.components["comp1"].names["pkg_config"] = "comp1"
        self.cpp_info.components["comp2"].names["pkg_config"] = "comp2"
EOF

cat >test_package/CMakeLists.txt <<EOF
cmake_minimum_required(VERSION 3.1)
project(test_package NONE)
include(FindPkgConfig)
pkg_search_module(COMP1 REQUIRED comp1>=0.1)
pkg_search_module(COMP2 REQUIRED comp2>=0.1)
EOF

cat >test_package/conanfile.py <<EOF
from conans import CMake, ConanFile
class A(ConanFile):
    build_requires = "pkgconf/1.7.3"
    generators = "pkg_config"
    def test(self):
        cmake = CMake(self)
        cmake.configure()
EOF
```

The following `.pc` files get created:
```
prefix=/home/maarten/.conan/data/some_package/0.1/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
libdir=${prefix}/lib
includedir=${prefix}/include

Name: some_package-comp2
Description: Conan package: some_package-comp2
Version: None
Libs: -L${libdir}
Cflags: -I${includedir}
```
and
```
prefix=/home/maarten/.conan/data/some_package/0.1/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
libdir=${prefix}/lib
includedir=${prefix}/include

Name: some_package-comp2
Description: Conan package: some_package-comp2
Version: None
Libs: -L${libdir}
Cflags: -I${includedir}
```

### Logs
```
----Running------
> cd '/home/maarten/programming/conan-problems/test_package/build/205c8b48087a3798d9a9666b887fcffad2ed516e' && cmake -G "Unix Makefiles" -DCONAN_IN_LOCAL_CACHE="OFF" -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY="ON" -DCONAN_EXPORTED="1" -Wno-dev '/home/maarten/programming/conan-problems/test_package'
-----------------
-- Found PkgConfig: /home/maarten/.conan/data/pkgconf/1.7.3/_/_/package/6af9cc7cb931c5ad942174fd7838eb655717c709/bin/pkgconf (found version "1.7.3") 
-- Checking for one of the modules 'comp1>=0.1'
CMake Error at /usr/share/cmake/Modules/FindPkgConfig.cmake:711 (message):
  None of the required 'comp1>=0.1' found
Call Stack (most recent call first):
  CMakeLists.txt:5 (pkg_search_module)


-- Checking for one of the modules 'comp2>=0.1'
CMake Error at /usr/share/cmake/Modules/FindPkgConfig.cmake:711 (message):
  None of the required 'comp2>=0.1' found
Call Stack (most recent call first):
  CMakeLists.txt:6 (pkg_search_module)


-- Configuring incomplete, errors occurred!
See also "/home/maarten/programming/conan-problems/test_package/build/205c8b48087a3798d9a9666b887fcffad2ed516e/CMakeFiles/CMakeOutput.log".

```

**Comment by memsharded** [comment]
This might be already fixed in https://github.com/conan-io/conan/pull/7524, merged in develop and ready for next Conan 1.29.

Could you please @madebr try to run from develop branch from sources to make sure?

**Comment by madebr** [comment]
Indeed, current develop creates correct comp1.pc pkg-config files.
```
prefix=/tmp/cc/.conan/data/some_package/0.1/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
libdir=${prefix}/lib
includedir=${prefix}/include

Name: some_package-comp1
Description: Conan package: some_package-comp1
Version: 0.1
Libs: -L${libdir}
Cflags: -I${includedir}
```
I wrongly assumed this bug would be fixed in yesterday's 1.28.2 release

**Comment by memsharded** [comment]
Nop, it could have probably been merged in 1.28.2, but it slipped. In any case Conan 1.29 should be out tomorrow, so hopefully it is not a big issue. Closing it, as this was fixed by #7524. Thanks for reporting!
